x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/edge/markers.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { useEuiTheme } from '@elastic/eui';
+
+const ArrowMarker = ({ id, color }: { id: string; color: string }) => {
+  return (
+    <marker
+      id={id}
+      markerWidth="14.4" // Increased by 20% from 12 to 14.4
+      markerHeight="14.4" // Increased by 20% from 12 to 14.4
+      viewBox="-12 -12 24 24" // Increased by 20% from "-10 -10 20 20" to "-12 -12 24 24"
+      markerUnits="strokeWidth"
+      orient="auto-start-reverse"
+      refX="0"
+      refY="0"
+    >
+      <polyline
+        strokeLinecap="round"
+        strokeLinejoin="round"
+        points="-6,-4.8 0,0 -6,4.8 -6,-4.8" // Adjusted by 20%
```

Comment:
Maybe for better maintainability, could we add it to a utility function to retrieve the points attribute as a list of (x,y) coordinates?

```js
// Arrow shape designed to form a symmetrical arrowhead:
const getArrowPoints = (width: number, height: number) => {
  return [
    [-width, -height],
    [0, 0],
    [-width, height],
    [-width, -height],
  ]
    .map(point => point.join(','))
    .join(' ');
};

// -6 shifts the arrowhead behind the tip at (0,0)
// ±4.8 sets the vertical spread of the arrowhead
const ARROW_POINTS = getArrowPoints(6, 4.8);

<polyline
        strokeLinecap="round"
        strokeLinejoin="round"
        points={ARROW_POINTS}
       // ...
/>
```